src/Objects/MonitorManager.vala

```diff
@@ -66,12 +66,22 @@ public class Display.MonitorManager : GLib.Object {
         virtual_monitors = new Gee.LinkedList<Display.VirtualMonitor> ();
         try {
             iface = Bus.get_proxy_sync (BusType.SESSION, "org.gnome.Mutter.DisplayConfig", "/org/gnome/Mutter/DisplayConfig");
-            iface.monitors_changed.connect (get_monitor_config);
+            iface.monitors_changed.connect (on_monitors_changed);
         } catch (Error e) {
             critical (e.message);
         }
     }
 
+    public void rescan_monitors () {
+        monitors.clear ();
+        virtual_monitors.clear ();
```

Comment:
Shouldn't you notify this change in virtual monitor size?

Reply:
Hmm, I'm not sure what you mean?

Reply:
I meant calling `notify["virtual-monitor-number"].  However, a better solution might be for the connections to be made directly to `virtual_monitors.size`  i.e. `monitor_anager.virtual_monitors.notify["size"].connect (() => ....` then no explicit notification would be needed.  However I would need to test this.

Reply:
I looked at resolving this review comment myself, and I actually don't see anything subscribing to virtual monitors' size. Maybe I am misunderstanding something?